pkg/operator/controller/ingress/controller.go

```diff
@@ -142,6 +145,16 @@ func enqueueRequestForOwningIngressController(namespace string) handler.EventHan
 						},
 					},
 				}
+			} else if ingressName, ok := labels[operatorcontroller.ControllerDeploymentLabel]; ok {
```

Comment:
this is the logic that adds reconcilation for pods (different label name), but same value.